.github/workflows/cicd.yml

```
@@ -19,10 +19,53 @@
+        run: echo "::notice::테스트 및 커버리지 리포트 생성이 완료되었습니다."
+
+  upload_coverage:
+    name: 📊 Codecov 업로드
+    needs: run_tests
+
+    # main으로 머지될때만 실행(커버리지 잘 나오는지 우선 dev일때도 되도록 수정)
+    if: github.event.pull_request.merged == true && (github.base_ref == 'main' || github.base_ref == 'dev')
+    runs-on: ubuntu-latest
+    steps:
+      - name: JaCoCo 리포트 다운로드
+        uses: actions/download-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)